plugins/package-managers/node/src/main/kotlin/Yarn.kt

```diff
@@ -80,7 +80,8 @@ class Yarn(
         // fixed minor version to be sure to get consistent results.
         checkVersion()
 
-    override fun runInstall(workingDir: File) = run(workingDir, "install", "--ignore-scripts", "--ignore-engines")
+    override fun runInstall(workingDir: File) =
+        run(workingDir, "install", "--ignore-scripts", "--ignore-engines", "--frozen-lockfile")
```

Comment:
Could you please use `--immutable` instead and add a link to https://yarnpkg.com/cli/install#options in the commit message, which states that `--frozen-lockfile` "will be removed in a later release"?